Makefile

```diff
@@ -414,6 +414,7 @@ dist/calicoctl-darwin-amd64: $(CALICOCTL_FILES) vendor
 
 dist/calicoctl-windows-amd64: $(CALICOCTL_FILES) vendor
```

Comment:
You should update the make target to match so that running `make dist/calicoctl-windows-amd64` twice doesn't rebuild the same binary.